langserver/integration_test.go

```diff
@@ -217,11 +337,74 @@ func integrationTest(
 		}
 	}
 
-	ctx := context.Background()
+	cfg := NewDefaultConfig()
+	cfg.UseBinaryPkgCache = true
```

Comment:
Sorry I haven't looked at this yet! But cc @slimsag about UseBinaryPkgCache question.

Reply:
The idea with `UseBinaryPkgCache` was that you are already running `go install` on your own. This concept actually comes from https://github.com/rogpeppe/godef -- which makes the same assumption. If you are not running `go install` on your code, however, then this option will not work well for you (you'll either get no results or stale results). This feature exists primarily because `go/loader` does not allow for caching type-checking results.

In your case here, I think you should just not set `UseBinaryPkgCache = true`. That means it will not use the cache, and will instead use the other codepath that performs type-checking on-the-fly (which is slower, but ideal for a test such as this)